packages/api/src/controllers/webhook.ts

```diff
@@ -287,9 +295,45 @@ const requestsFieldsMap: FieldsMap = {
   statusCode: `webhook_response.data->'response'->>'status'`,
 };
 
+app.put("/:id/log/:requestId/resend", authorizer({}), async (req, res) => {
```

Comment:
Should this be a `POST` instead? Unless we somehow keep the same `requestId` when we resend, but I guess we don't 🤔

Reply:
So you're suggesting `POST /:id/log/resend` with the requestID in the request body?

Reply:
No, I think `POST /:id/log/:requestId/resend` is fine! I just that a `PUT .../:requestId` would seem like it's updating the same request/log, instead of creating a new one. Also I think the general pattern for "actions" (in this case "resend") is to be a `POST` as well